libs/features/branch-quick-links/branch-quick-links.js

```diff
@@ -0,0 +1,81 @@
+import { getMetadata, loadStyle, getConfig, createTag } from '../../utils/utils.js';
+
+function addLoader(elem, text) {
+  const { base } = getConfig();
+  loadStyle(`${base}/features/branch-quick-links/branch-quick-links.css`);
+  const lbl = createTag('div', { class: 'pbar-label' });
```

Comment:
just naming this `label` would help with comprehension compared to `lbl` u save 2 characters, but its not immediately obvious further down in the function on what `lbl` is.